plone/app/upgrade/v60/configure.zcml

```diff
@@ -82,9 +82,14 @@
         destination="6004"
         profile="Products.CMFPlone:plone">
 
+      <gs:upgradeDepends
+          title="Run to6004 upgrade profile."
+          import_profile="plone.app.upgrade.v60:to6004"
```

Comment:
Should these two upgrade steps be switched around? In the current order, the `plone-logged-in` bundle is added by the profile, but later deleted, presumably by the next upgrade step.